elementtypes/TranslateElementType.php

```diff
@@ -243,8 +243,7 @@ public function getIndexHtml($criteria, $disabledElementIds, $viewState, $source
     {
         // If the site only has 1 locale enabled, set the translated locale to the primary (and only) locale
         if(empty($criteria['locale'])) {
-            $localization = new LocalizationService();
-            $criteria['locale'] = $localization->getPrimarySiteLocale();
+            $criteria['locale'] = craft()->i18n->getPrimarySiteLocale()
```

Comment:
My bad. The tests aren't included in the Craft build anymore so not sure how I would run them...